draft-ietf-modpod-group-processes.md

```diff
@@ -462,6 +482,10 @@ These individuals contributed additional improvements:
 {:aside}
 > RFC Editor: Please remove this appendix before publication.
 
+## Since draft-ietf-modpod-group-processes-07
```

Comment:
Please add a changelog entry?

Reply:
There is one.I think there is one?

Reply:
The only one the `Pete Resnick issues and similar` one below. Is that also capturing the changes in this PR?